ui/src/Tags/PersonTag/PersonTagClient.test.tsx

```diff
@@ -0,0 +1,102 @@
+    const basePersonTagsUrl = `/api/spaces/${spaceUuid}/person-tags`;
+    const expectedConfig = {
+        headers: {
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer 123456',
+        },
+    };
+    const cookies = new Cookies();
+
+    beforeEach(() => {
+        cookies.set('accessToken', '123456');
+
+        // @ts-ignore
+        Axios.post = jest.fn(x => Promise.resolve({
+            data: 'Created Person Tag',
+        } as AxiosResponse));
+        // @ts-ignore
+        Axios.put = jest.fn(x => Promise.resolve({
+            data: 'Updated Person Tag',
+        } as AxiosResponse));
+        // @ts-ignore
+        Axios.delete = jest.fn(x => Promise.resolve({
+            data: 'Deleted Person Tag',
+        } as AxiosResponse));
+        // @ts-ignore
+        Axios.get = jest.fn(x => Promise.resolve({
+            data: 'Get Person Tags',
+        } as AxiosResponse));
+    });
+
+    afterEach(() => {
+        cookies.remove('accessToken');
+    });
+
+    it('should return all person tags for space', function(done) {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        PersonTag.get(spaceUuid!)
+            .then((response) => {
+                expect(Axios.get).toHaveBeenCalledWith(basePersonTagsUrl, expectedConfig);
+                expect(response.data).toBe('Get Person Tags');
+                done();
+            });
+
+    });
+
+    it('should create a person tag and return that person tag', function(done) {
+        const expectedPersonAddRequest = { name: TestUtils.personTag1.name };
+        PersonTag.add(expectedPersonAddRequest, TestUtils.space)
+            .then((response) => {
+                expect(Axios.post).toHaveBeenCalledWith(basePersonTagsUrl, expectedPersonAddRequest, expectedConfig);
+                expect(response.data).toBe('Created Person Tag');
+                done();
+            });
+    });
+
+    it('should edit a person tag and return that person tag', function(done) {
+        const expectedPersonEditRequest = {
+            id: TestUtils.personTag1.id,
+            name: TestUtils.personTag1.name,
+        };
+        PersonTag.edit(expectedPersonEditRequest, TestUtils.space)
+            .then((response) => {
+                expect(Axios.put).toHaveBeenCalledWith(`${basePersonTagsUrl}/${TestUtils.personTag1.id}`, expectedPersonEditRequest, expectedConfig);
+                expect(response.data).toBe('Updated Person Tag');
+                done();
+            });
+    });
+
+    it('should delete a person tag', function(done) {
+        const expectedUrl = `${basePersonTagsUrl}/${TestUtils.personTag1.id}`;
+        PersonTag.delete(TestUtils.personTag1.id, TestUtils.space)
+            .then((response) => {
+                expect(Axios.delete).toHaveBeenCalledWith(expectedUrl, expectedConfig);
+                expect(response.data).toBe('Deleted Person Tag');
+                done();
+            });
```

Comment:
On delete, should we verify that anyone who was assigned that tag now has 1 less tag? (That may have already been done in another test in another file.)